client/data/marketplace/use-es-query.ts

```diff
@@ -161,7 +161,7 @@ export const useESPluginsInfinite = (
 			};
 		},
 		getNextPageParam: ( lastPage ) => {
-			return lastPage.data.page_handle || undefined;
+			return lastPage?.data?.page_handle || undefined;
```

Comment:
You can remove the `undefined` since the return value will be `undefined` anyway if any of the accessed object paths is nullish.

```suggestion
			return lastPage?.data?.page_handle;
```

Reply:
That makes sense. Thank you!